### DISCUSSION TITLE
Outdated TurnKey Nextcloud Version - how to upgrade?

### DISCUSSION THREAD

**Discussion by speedAmaster** [tteck/Proxmox#1929] [Help Needed]
I used the TurnKey Nextcloud helper script to setup my Nextcloud on my proxmox - long tiome ago......

Now it looks like, I am heavily outdated (Debian 10, PHP7.3.31, Nextcloud 23.0.12 (even it says "Deine Version ist aktuell)).

Is there any hint to updated from TurnKey? Any particular order (1st PHP from 7.3 to 7.4 to 8.x)? How do I perform that in TurnKey-Container on Proxmox?

Which Linux distribution are you employing?
     Debian 10 (TurnKey GNU/Linux 16.1 (Debian 10/Buster))

**Comment by tteck** [comment]
I don't _think_ that there is a way to upgrade an existing TurnKey appliance.
BTW, the TurnKey helpers were created on 2023-08-30, not that long ago.

**Comment by lukyjay** [comment]
There is some documentation here: https://www.turnkeylinux.org/docs/appliance-upgrade

In short, you can try the package manager but to upgrade the container you really need to redeploy and restore a backup (similar to Docker).

**Comment by Smile2Grow** [comment]
You can update Turnkey Nextcloud in the administration area.
